### ISSUE TITLE
Add focus outlines to controls

### ISSUE THREAD

**Issue by chrisauclair** [videojs/video.js#3200]
The default theme should include clear focus outlines on the controls by default, whether they're something custom or from the user agent. I know that designers love to hate them, but turning them off in the player by default is like turning them off in a browser by default. Leaving them enabled will steer most designers and devs in the right direction, and the rest can choose to weigh the implications of removing them.

**Comment by gkatsev** [comment]
We do have a focus highlight outline thing replacement. Though, it's possibly should be more vibrant to make it more easily visible and noticeable.

**Comment by chrisauclair** [comment]
@gkatsev , the focus glow lacks enough contrast to be a suitable replacement for the outline. If the outline is such a non-starter, an alternative might be a very strong background color that is not the same color as the icon (or the same color as anything else in the player). 

[I strongly recommend this brief, opinionated read on the subject.](http://www.outlinenone.com/)

**Comment by OwenEdwards** [comment]
@chrisauclair :+1:

**Comment by gkatsev** [comment]
We should probably enable this as part of 6.0.

**Comment by gkatsev** [comment]
Just wanted to ping this thread directly: outlines are restored in 6.0

**Comment by thijstriemstra** [comment]
@gkatsev how to disable them across all player elements?
